git.go

```diff
@@ -112,12 +112,12 @@ func (g *Gitter) TreeHash(gitRev string, pkgPath string) (string, error) {
 	}
 }
 
-var lsRemoteRegex = regexp.MustCompile(`^([0-9a-f]+)\s+HEAD$`)
+var lsRemoteRegex = regexp.MustCompile(`^([0-9a-f]+)\s+[^ ]*$`)
```

Comment:
Is that a space literal?  Could you just replace the `[^ ]*` with `\S*` (or even `\S+`)?

Reply:
Good call. `\S+` it is